examples/advanced/reconstruct_python.py

```diff
@@ -8,10 +8,15 @@
 
 """
 
-from lark import Token
+from lark import Lark, Token
 from lark.reconstruct import Reconstructor
 
-from python_parser import python_parser3
+from python_parser import PythonIndenter
+
+# Official Python grammar by Lark
+python_parser3 = Lark.open_from_package('lark', 'python.lark', ['grammars'], 
```

Comment:
Do we need `.lark`?

Reply:
What do you mean?

Reply:
Do we need to manually add the extension `.lark`